app/models/katello/repository.rb

```diff
@@ -1050,6 +1054,12 @@ def remove_docker_content(manifests)
       DockerMetaTag.cleanup_tags
     end
 
+    def allow_updates?
+      # allow the update if this repo is not in the default view
+      return true unless in_default_view?
+      root.allow_updates?(ALLOWED_UPDATE_FIELDS)
```

Comment:
It's a little bit confusing that a root method is being called to check instance repo attributes, but the only alternatives are to either duplicate code or make some separate helper file, so I think I prefer this.